src/libclang/libclang_parser.cpp

```diff
@@ -243,11 +248,30 @@ cppast::libclang_compile_config::libclang_compile_config(
                 add_flag(std::move(flag));
             }
             else if (flag == "-std")
+            {
                 // standard
+                try
```

Comment:
I don't like the use of try-catch here. I think it's better if to_standard uses `type_safe::optional`.

Also: if `to_standard` is only used here, I think it's better if it's an implementation detail of the file and not part of the public API.